object-manipulation-1/src/get-values.ts

```diff
@@ -0,0 +1,22 @@
+/* exported getValues */
+
+interface Jobs {
```

Comment:
Since this describes just a single job, it would be best to name it with a singular noun: `Job`.